.github/workflows/docker-tag.yml

```diff
@@ -0,0 +1,28 @@
+name: Tag new images version
+on:
+  workflow_dispatch:
```

Comment:
Should we plan a cron?

Reply:
Sounds good. You proposed quarterly, right?

Reply:
@PerfectSlayer Sounds good, roughly aligned with JDK CPU dates. Feel free to tweak the cron.